app/views/badges/require.slim

```diff
@@ -3,13 +3,13 @@
     .col-md-12
       .form-container
         h3
-          | OBTER BADGE_
+          | OBTER #{t('badge').upcase.tr('á','Á')}_
```

Comment:
@frmendes Somehow the `upcase` method does not transform lowercase accented characters to uppercase ones... But i will move it into a method 👍